--- origsrc/at-spi2-core-2.6.3/atspi/atspi-action.c	2012-12-04 23:53:45.000000000 -0600
+++ src/at-spi2-core-2.6.3/atspi/atspi-action.c	2012-12-16 22:21:45.976848300 -0600
@@ -89,7 +89,7 @@ atspi_action_get_description (AtspiActio
  *        keybinding string is the "keyboard shortcut" which invokes the 
  *        action without posting any menus. 
  *        Meta-keys are indicated by the conventional strings
- *        "<Control>", "<Alt>", "<Shift>", "<Mod2>",
+ *        "&lt;Control&gt;", "&lt;Alt&gt;", "&lt;Shift&gt;", "&lt;Mod2&gt;",
  *        etc. (we use the same string as gtk_accelerator_name() in 
  *        gtk+-2.X.
  *
